app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormatExpr.scala

```diff
@@ -160,7 +160,67 @@ object RoundingMode {
   )
 }
 
-sealed trait TextConstraint
+sealed trait TextConstraint {
+  val defaultCssClassName: String = this match {
+    case Number(maxWholeDigits, maxFractionalDigits, _, _) =>
+      val size =
+        if (maxFractionalDigits > 0)
+          maxWholeDigits + maxFractionalDigits + 1
+        else
+          maxWholeDigits
+
+      deriveCssClassNameForNumber(size)
+    case PositiveNumber(maxWholeDigits, maxFractionalDigits, _, _) =>
+      val size =
+        if (maxFractionalDigits > 0)
+          maxWholeDigits + maxFractionalDigits + 1
+        else
+          maxWholeDigits
```

Comment:
Let's push this logic down to get rid of duplication from lines 166-170.

(if appropriate length identifiers would be used whole `match` expression will became nicely aligned)
```
case Number(whole, fractional, _, _)         => deriveCssClassNameForNumber(whole, fractional)
case PositiveNumber(whole, fractional, _, _) => deriveCssClassNameForNumber(whole, fractional)
case BasicText                               => CssClassSize._20
...
```